polyfill/lib/casts.mjs

```diff
@@ -37,7 +37,7 @@ export function CastAbsolute(arg, aux) {
   if ('bigint' === typeof arg) return new Absolute(arg);
   if ('string' === typeof arg) {
     try {
-      return Absolute.fromString(arg);
+      return Absolute.from(arg);
```

Comment:
Instead, please move the string handling into the `CastFoo` functions, and make `Foo.from` a thin wrapper over `CastFoo`.

Reply:
@Ms2ger that's close to what I propose in #269.